python/lsst/ts/externalscripts/auxtel/wep_checkout.py

```diff
@@ -0,0 +1,173 @@
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            $id: https://github.com/lsst-ts/ts_externalscripts/wep_checkout.yaml
+            title: WepCheckout
+            description: Configuration schema for WepCheckout.
+            type: object
+            properties:
+              intra_visit_id:
+                description: >
+                  An intra focus visit id.
+                type: string
+                default: "2021110400954"
+              extra_visit_id:
+                description: >
+                  An extra focus visit id.
+                type: string
+                default: "2021110400955"
+              dz:
+                description: >
+                  Offset for the intra/extra images.
+                type: number
+                default: 1.5
+              side:
+                description: >
+                  Set stamp size for WFE estimation.
+                type: number
+                default: 192
+              expected_zern_defocus:
+                description: >
+                  Expected Zernike defocus coefficient.
+                type: number
+                default: 69.856
+              expected_zern_coma_x:
+                description: >
+                  Expected Zernike coma_x coefficient.
+                type: number
+                default: 35.745
+              expected_zern_coma_y:
+                description: >
+                  Expected Zernike coma_y coefficient.
+                type: number
+                default: 70.311
+              threshold:
+                description: >
+                  Tolerance for Zernike coefficients.
+                type: number
+                default: 20
+            additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    async def configure(self, config):
+
+        self.intra_visit_id = config.intra_visit_id
+        self.extra_visit_id = config.extra_visit_id
+        self.dz = config.dz
+
+        self.expected_zernikes = {
+            "defocus": config.expected_zern_defocus,
+            "coma_x": config.expected_zern_coma_x,
+            "coma_y": config.expected_zern_coma_y,
+        }
+
+        self.threshold = config.threshold
+
+        # 192 pix is size for dz=1.5, but gets automatically
+        # scaled based on dz later, so can multiply by an
+        # arbitrary factor here to make it larger
+        self.side = config.side * 1.1  # normally 1.1
+
+        self.config = config
+
+    def set_metadata(self, metadata):
+        metadata.duration = 10.0
+
+    async def run(self):
+        """Run the WEP pipeline checkout."""
+        self.log.info("Starting WEP pipeline checkout.")
+
+        # Process images using WEP
+        donut_diameter = int(np.ceil(self.side * self.dz / 1.5 / 2.0) * 2)
+
+        try:
+            intra_result, extra_result, wep_results = await run_wep(
```

Comment:
Note that [run_wep](https://github.com/lsst-ts/ts_externalscripts/blob/acff9e0ef108818519ff8562367c9596564e4515/python/lsst/ts/externalscripts/auxtel/latiss_wep_align.py#L141) is not a coroutine, but a regular method and cannot be awaited. 

You will have to run it in a `ProcessPoolExecutor` like the wep script does to make sure it does not block the event loop. 

See how this is done [here](https://github.com/lsst-ts/ts_externalscripts/blob/acff9e0ef108818519ff8562367c9596564e4515/python/lsst/ts/externalscripts/auxtel/latiss_wep_align.py#L107-L129).